src/devices/DCMotor/DCMotor.cs

```diff
@@ -41,58 +41,130 @@ public virtual void Dispose()
         }
 
         /// <summary>
-        /// Constructs DCMotor class.
-        /// Depending on the settings it will pick implementation suitable for the given connection.
+        /// Creates DCMotor instance which allows to control speed in one direction.
         /// </summary>
-        public static DCMotor Create(DCMotorSettings settings)
+        /// <param name="speedControlChannel">PWM channel used to control the speed of the motor</param>
+        /// <returns>DCMotor instance</returns>
+        public static DCMotor Create(PwmChannel speedControlChannel)
```

Comment:
I suppose the idea of having factory methods is to hide from people which type (how many pins) of DCMotor are they getting?

Reply:
it's an implementation detail so yes